test-site/src/ts/initDirectAnswers.ts

```diff
@@ -49,6 +49,9 @@ const FIELD_VALUE_QUERIES = {
   ],
   [BuiltInFieldType.Integer]: [
     'oliver shi outdoor pools?' // number
+  ],
+  [BuiltInFieldType.Phone]: [
+    'oliver shi phone?' // string
```

Comment:
can we add "amani farooque phone number" here? it's kind of a classic

Reply:
okay i'll add it XD